package/fuseki/package

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# Copyright (c) 2025 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(fuseki)
+## This should be a short description targeted at users of your package
+# Start directly with a noun or a verb, do not include “for the reMarkable”
+pkgdesc="Go frontend using GnuGo"
+## URL to where the project can be found on the Internet
+url="https://github.com/palfrey/fuseki"
+## Version of the package (see below)
+pkgver=0.0.1-1
+## ISO-8601 timestamp at which the source of this version was released
+timestamp=2025-08-25T16:56+0100
+## Section that best matches this package
+# See <https://github.com/toltec-dev/toltec/blob/stable/docs/package.md#section>
+# for available sections
+section=games
+## This should be your contact information
+maintainer="Tom Parker-Shemilt <palfrey@tevps.net>"
+## A license under which the upstream source is available
+# Choose among the list of valid SPDX license identifiers
+# <https://spdx.org/licenses/>
+license=AGPL-3.0-or-later
+archs=(rm1)
```

Comment:
Since you are using libremarkable, and don't appear to be doing anything rM1 specific in your codebase, I don't see a reason to make this rmall, I'll be testing on both rM1 and rM2 anyways before this gets merged.